src/pages/HomePage/index.tsx

```diff
@@ -58,14 +78,38 @@ export default function HomePage() {
     setCurrentPage(currentPage - 1);
   };
 
+  const handleClickSearch = async () => {
+    if (isAllowedToSearch && searchText) {
+      const result = await searchTransactionQuery({
+        variables: { transaction: searchText, address: searchText },
+      });
+
+      if (result.data?.transaction?.id) {
+        window.location.assign(`/transaction/${searchText}`);
+      } else if (result.data?.transactionsByOwner?.edges?.length) {
+        window.location.assign(`/address/${searchText}`);
+      } else {
+        setIsSearchNotFound(true);
+        setTimeout(() => setIsSearchNotFound(false), 1500);
```

Comment:
Why we need this timeout?

Reply:
to show "Not found" message, then hide it after `1500 ms`